### PR TITLE
Create UnmarkAttend command

### LINKED ISSUE #174: Add unmark attendance command

### PR DESCRIPTION
**Pull request by amitha2210**
Closes #174 

1. Add new Command and Parser UnmarkAttendanceByStudentCommand
2. Add test cases for UnmarkAttendanceByStudentCommand and relevant classes

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ * Marks the attendance of a person identified using it's displayed index from the addres book as absent.
+ */
+public class UnmarkAttendanceByStudentCommand extends Command {
+
+    public static final String COMMAND_WORD = "umas";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Unmarks the attendance of the student identified "
+            + "by the index number used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_ATTENDANCE + "ATTENDANCE]"
```
Comment (Slipperybara) [inline src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java:31]: Think by UG standard, including square bracket refers to the field being optional(eg. inside editCommand)
Not too sure if we are going to apply this standard here also, we can discuss about it

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        }
+
+        Person studentToUnmarkAttendance = lastShownList.get(targetIndex.getZeroBased());
+        Participation currentParticipation = model.getParticipationList().stream()
+                .filter(participation -> participation.getStudent().equals(studentToUnmarkAttendance)
+                        && participation.getTutorial().equals(this.tutorial)).findFirst()
+                .orElseThrow(() -> new CommandException(String.format(MESSAGE_INVALID_TUTORIAL_FOR_STUDENT, tutorial)));
+
+        List<Attendance> updatedAttendance = new ArrayList<>(currentParticipation.getAttendanceList());
+        updatedAttendance.add(attendance);
```
Comment (Slipperybara) [inline src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java:75]: Here should be removing attendance?

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java (hunk 1 above)
Comment (amitha2210) [reply src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java:31]: Good observation, was following the standard for the other commands relating to attend, but will edit this for for all them. Thanks!

File: src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java (hunk 2 above)
Comment (gladwintan) [reply src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java:75]: Yea I think we should remove the attendance here and throw an exception if the attendance for that date is not found

Comment (amitha2210) [reply src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java]: Thanks for the catch! Fixed now.

**Review by Slipperybara — APPROVED** [review]
LGTM

**Review by gladwintan — APPROVED** [review]
LGTM

**Pull request merged**